appcenter/android/src/main/java/com/microsoft/appcenter/reactnative/appcenter/ReactNativeUtils.java

```diff
@@ -111,11 +110,6 @@ private static Object toObject(@Nullable ReadableMap readableMap, String key) {
 
     private static Date tryToGetDate(@Nullable ReadableMap readableMap) {
         Map<String, Object> map = toMap(readableMap);
-        //it is not allowed that there should be more than one date at a time for Custom Properties
```

Comment:
This change doesn't look correct. We either remove the whole method, or keep this logic. Let's check what "Custom Properties" is meant by this commen. And check if we can remove this method whole.

Reply:
This method is using in Analytics for trackEvent. I'll revert these changes.